src/main/java/com/thealgorithms/sorts/BeadSort.java

```diff
@@ -1,40 +1,53 @@
 package com.thealgorithms.sorts;
 
-// BeadSort Algorithm(wikipedia) : https://en.wikipedia.org/wiki/Bead_sort
-// BeadSort can't sort negative number, Character, String. It can sort positive number only
+import java.util.Arrays;
 
+/**
+ * BeadSort cannot sort negative numbers, characters, or strings.
+ * It only works for non-negative integers.
```

Comment:
```suggestion
 * BeadSort works for non-negative integers only.
```

Reply:
Looks like this is already included into main description for sort method in javadocs